lib/components/viewers/route-details.tsx

```diff
@@ -149,12 +166,11 @@ class RouteDetails extends Component<Props> {
             <HeadsignSelectLabel htmlFor="headsign-selector-label">
               <FormattedMessage id="components.RouteDetails.stopsTo" />
             </HeadsignSelectLabel>
-            <SortResultsDropdown
+            <PatternSelectDropdown
```

Comment:
If we decide to leave the default margins (which would mean a slightly taller dropdown) just flagging it would probably be good to throw an `align-items: center` on the parent here so that this row doesn't look uneven: 
<img width="405" alt="image" src="https://github.com/user-attachments/assets/7b53bce9-1cba-479f-a67b-6a50f099fd94">

Reply:
This is the only place I really had a problem with the way the new styling looks. If we can get this vertically centered then I'm happy to approve the Percy diff

Reply:
Take a look a [78d52eb](https://github.com/opentripplanner/otp-react-redux/pull/1251/commits/78d52ebb5f64733579426928fae93b9e8086526b). This addresses @amy-corson-ibigroup and @daniel-heppner-ibigroup suggestions!

Reply:
Looking really good, I would maybe add some `margin-bottom` to this `div` just to add a little vertical space between the selector and the "stops in this direction of travel": 
![image](https://github.com/user-attachments/assets/00950050-88fd-4d3c-878e-843d345ecab8)

Reply:
added here! [78d52eb](https://github.com/opentripplanner/otp-react-redux/pull/1251/commits/78d52ebb5f64733579426928fae93b9e8086526b)